client/planning-extension/src/assignments-overview/index.tsx

```diff
@@ -0,0 +1,245 @@
+        );
+    }
+
+    handleContentChanges(changes: Array<IResourceChange>) {
+        const state = this.state;
+
+        if (state.loading === true) {
+            return;
+        }
+
+        const {assignments} = state;
+
+        const refetchDesks = changes.find(({resource}) => resource === 'desks');
+        const refetchContentTypes = changes.find(
+            ({resource, itemId}) => resource === 'vocabularies' && itemId === 'g2_content_type',
+        );
+        const refetchAssignments = changes.find(
+            ({changeType, resource, itemId}) =>
+                (resource === 'assignments' && (changeType === 'created' || changeType === 'deleted'))
+                || (
+                    resource === 'assignments'
+                    && changeType === 'updated'
+                    && assignments.find(({_id}) => _id === itemId) != null
+                ),
+        );
+
+        Promise.all([
+            refetchDesks ? fetchDesks() : Promise.resolve(state.desks),
+            refetchContentTypes ? fetchContentTypes() : Promise.resolve(state.contentTypes),
+            refetchAssignments ? fetchAssignments(state.currentUser._id) : Promise.resolve(state.assignments),
+        ]).then(([desks, contentTypes, assignments]) => {
+            this.setState({
+                loading: false,
+                desks: desks,
+                contentTypes: contentTypes,
+                assignments: assignments,
+            });
+        });
+    }
+
+    componentDidMount() {
+        superdesk.session.getCurrentUser()
+            .then((currentUser) =>
+                Promise.all([
+                    fetchDesks(),
+                    fetchContentTypes(),
+                    fetchAssignments(currentUser._id),
+                ]).then(([desks, contentTypes, assignments]) => {
+                    this.setState({
+                        loading: false,
+                        currentUser: currentUser,
+                        desks: desks,
+                        contentTypes: contentTypes,
+                        assignments: assignments,
+                    });
+                })
+            );
+    }
+
+    componentWillUnmount() {
+        this.eventListenersToRemoveBeforeUnmounting.forEach((removeListener) => {
+            removeListener();
+        });
+    }
+
+    render() {
+        if (this.state.loading === true) {
+            return null;
+        }
+
+        const {assignments, desks, contentTypes} = this.state;
+        const itemsCount = assignments.length;
+        const grouped = groupBy(assignments, (item) => item.assigned_to.desk);
+
+        return (
+            <DropdownTree
+                groups={Object.keys(grouped).map((deskId) => ({
+                    render: () => (
+                        <GroupLabel>
+                            <Badge type="highlight" text={grouped[deskId].length.toString()} />
+                            <span style={{marginLeft: 6}}>{desks[deskId].name}</span>
```

Comment:
Would a margin left of 8 pixels work? (in accordance with https://ui-framework.superdesk.org/#/design/layout-principles)
If so, it might be beneficial to use the classname `sd-margin-l--1`